source/slang/slang-parser.cpp

```diff
@@ -229,6 +229,29 @@ namespace Slang
                 lastErrorLoc = loc;
             }
         }
+
+    public:
+        void setBindingOffset(int binding, int64_t byteOffset)
+        {
+            bindingToByteOffset.set(binding, byteOffset);
+        }
+        const int64_t getNextBindingOffset(int binding) 
```

Comment:
As I explained it before, const on a primitive type doesn't do anything here.
It can be easily copied by a value to non-const variable implicitly by the compiler.